### PR TITLE
gh-528: Split configure file in readable chunks

### PR DESCRIPTION
**Pull request by bobvanluijt**
Splitted the config file in multiple files.

### CODE REVIEW

File: restapi/configure_weaviate_things_helpers.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ * Contact: @CreativeSofwFdn / bob@kub.design
+ */
+
+// Package restapi with all rest API functions.
+package restapi
+
+import "math"
+
+// getLimit returns the maximized limit
+func getLimit(paramMaxResults *int64) int {
```
Comment (moretea) [inline restapi/configure_weaviate_things_helpers.go]: This is not specific for things, is it?

File: restapi/configure_weaviate_things_helpers.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	if paramMaxResults != nil {
+		maxResults = *paramMaxResults
+	}
+
+	// Max results form URL, otherwise max = config.Limit.
+	return int(math.Min(float64(maxResults), float64(serverConfig.Environment.Limit)))
+}
+
+// getPage returns the page if set
+func getPage(paramPage *int64) int {
```
Comment (moretea) [inline restapi/configure_weaviate_things_helpers.go]: This is not specific for things.

File: restapi/configure_weaviate_keys_helpers.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+		keyResponse := models.KeyGetResponse{}
+
+		// Get the key to delete
+		dbConnector.GetKey(ctx, keyID, &keyResponse)
+
+		databaseConnector.DeleteKey(ctx, &keyResponse.Key, keyID)
+	}
+}
+
+func generateMultipleRefObject(keyIDs []strfmt.UUID) models.MultipleRef {
```
Comment (moretea) [inline restapi/configure_weaviate_keys_helpers.go]: Is this specific to keys?

**Comment by moretea** [comment]
This is a pretty intrusive change w.r.t. changed lines. Given that Git is a stupid content tracker, this might break a few of my branches. That is why I started doing this for new endpoints, and not existing endpoints. Could we coordinate on these things next time please?

File: restapi/configure_weaviate_things_helpers.go (hunk 1 above)
Comment (bobvanluijt) [reply restapi/configure_weaviate_things_helpers.go]: They are added to these files based on where they are used. But off course, can be moved to the general helper.

File: restapi/configure_weaviate_things_helpers.go (hunk 2 above)
Comment (bobvanluijt) [reply restapi/configure_weaviate_things_helpers.go]: They are added to these files based on where they are used. But off course, can be moved to the general helper.

**Comment by bobvanluijt** [comment]
Is that so? It is simply the same package split up into files for readability.

Of course, large structural changes should be done coordinated consensus-based.

**Pull request merged**